bosch_locator_bridge/CMakeLists.txt

```diff
@@ -111,8 +113,12 @@ target_include_directories(${PROJECT_NAME}_server_node PUBLIC
   $<INSTALL_INTERFACE:include/${PROJECT_NAME}>)
 target_link_libraries(${PROJECT_NAME}_server_node  ${Boost_LIBRARIES})
 ament_target_dependencies(${PROJECT_NAME}_server_node
+  tf2_geometry_msgs
+  tf2_ros
   rclcpp
+  sensor_msgs
   std_srvs
+  pcl_conversions
```

Comment:
These dependencies are not needed for the server node, right?

Reply:
I did not build without adding them, because the server node was compiling `rosmsgs_datagram_converter.cpp`, which is actually not needed.